### ISSUE TITLE
Memory Leak

### ISSUE THREAD

**Issue by ratanaklun** [TimelordUK/node-sqlserver-v8#29]
I've been trying to find the cause of a memory leak in my application that uses msnodesqlv8 and it seems to be caused by msnodesqlv8. I have the following test code:
```
const msnodesqlv8 = require('msnodesqlv8');

msnodesqlv8.open('Driver={SQL Server Native Client 11.0};Server=localhost\\SQLEXPRESS;Trusted_Connection=yes;', (err, conn) => {
	if (err) {
		throw err;
	}
	setInterval(() => {
		conn.queryRaw('select 1+1', (err, results) => {
			if (err) {
				throw err;
			}
			console.log(results);
		});
	}, 100);
});
```
The query executes successfully, but when monitoring the memory usage, the usage just keeps going up until it uses so much the process has to be killed. It does not look like queries are being cleaned up properly.

I took a look into the code and think it might have to do with SQL handles not being freed. I noticed that in the `Free` method of the `OdbcHandle` class, `handle` is set to `nullptr` before the call to `SQLFreeHandle`. This would cause the handle to not be freed. ([Reference](https://github.com/TimelordUK/node-sqlserver-v8/blob/890fde5/src/OdbcHandle.cpp#L60))

**Comment by TimelordUK** [comment]
Thanks for letting me know I will take a look and get back to you promptly

**Comment by TimelordUK** [comment]
your test was a great help, thanks.  This issue should now be corrected in latest version 0.2.11.

Please give it a go and see if it helps.

**Comment by ratanaklun** [comment]
Thanks for getting on this issue quickly. Yes, it does seem a lot more stable. Closing for now, but I'll let you know if I experience any issues.
